common/compact_uint_test.go

```diff
@@ -23,8 +23,14 @@ import (
 )
 
 func TestCompactUint(t *testing.T) {
-	a := uint64(1200000)
+	a := uint64(120)
 	compactValue := SetCompactUint(a)
 	unCompactValue, _ := GetCompactUint(compactValue)
 	assert.Equal(t, unCompactValue, a)
+
+	a = uint64(12000000)
+	compactValue = SetCompactUint(a)
+	assert.Panics(t, func() {
+		GetCompactUint(compactValue)
```

Comment:
what‘s  this?

Reply:
i don't know, the compact_utils.go is not used in project.

Reply:
I mean GetCompactUint panic means it is buggy.